src/Extractor/FromSourceMappingExtractor.php

```diff
@@ -61,35 +61,11 @@ public function getPropertiesMapping(MapperGeneratorMetadataInterface $mapperMet
                 continue;
             }
 
-            $sourceTypes = $this->propertyInfoExtractor->getTypes($mapperMetadata->getSource(), $property);
```

Comment:
this code (and their equivalent in `FromTargetMappingExtractor` / `SourceTargetMappingExtractor`) have been moved in a `TransformerResolver`